tests/benchmarks/spi_endless/boards/nrf54l15dk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
Support for PDK can be remove for this tests - no need to keep pdk files, only dk can stay.

Reply:
done

Reply:
It looks liek pdk overlay still exist, that one is new, not renamed.

Reply:
done